### DISCUSSION TITLE
It looks strange of the relationship between the  wind power generation and wind speed in Kenya

### DISCUSSION THREAD

**Discussion by Haiping007** [pypsa-meets-earth/pypsa-earth#982] [Q&A]
I did some investigation on the relationship between wind power generation and wind speed in Kenya. I found it seemed strange and did not follow the rules of wind energy.

A typical curve of wind power generation and wind speed of a wind turbine is presented in Fig.1. When the wind speed is in a range (such as 4-13m/s),  the wind power generation increases with the wind speed increases. But there seem no any relationships shown  in Fig.2 (bus KE07) and Fig.3 (bus KE00). We are wondering if there is something wrong?

![image](https://github.com/pypsa-meets-earth/pypsa-earth/assets/146131420/582fe094-e27b-417b-879b-2f1ca515f7b9)

![wnd100m_Kenya_2013_KE07](https://github.com/pypsa-meets-earth/pypsa-earth/assets/146131420/7ec420b5-3bbf-43a9-bd93-a5d1c4cffe63)
![wnd100m_Kenya_2013_KE00](https://github.com/pypsa-meets-earth/pypsa-earth/assets/146131420/1680c4c0-598f-4579-afad-c2e9854407ca)

**Comment by ekatef** [comment]
> I did some investigation on the relationship between wind power generation and wind speed in Kenya. I found it seemed strange and did not follow the rules of wind energy.
> 
> A typical curve of wind power generation and wind speed of a wind turbine is presented in Fig.1. When the wind speed is in a range (such as 4-13m/s), the wind power generation increases with the wind speed increases. But there seem no any relationships shown in Fig.2 (bus KE07) and Fig.3 (bus KE00). We are wondering if there is something wrong?

Hello @Haiping007!

Thanks a lot for sharing the results of the test. Basically, the wind speed in the cell is transformed into the wind potential applying a logarithmic profile for the wind speed and a working curve for a selected wind turbine. This transformation is implemented [here](https://github.com/pypsa-meets-earth/pypsa-earth/blob/283668a1c0be438ae0feb910c92ea72e92069c95/scripts/build_renewable_profiles.py#L741C21-L755) where `func( )` calls a method defined by [`atlite` package](https://github.com/PyPSA/atlite/blob/master/atlite/convert.py) for a particular renewable technology.

So, there definitely should be a correlation between the wind speed and wind potential. Not sure what went wrong for the check which results you shared. If possible, could you please share the code you have used for this test?

**Reply by Haiping007** [reply]
Yes, sure, shall I share it here or send you by email?

**Reply by ekatef** [reply]
> Yes, sure, shall I share it here or send you by email?

As you wish :) Any would work.
